README.md

```diff
@@ -1,37 +1,41 @@
 # FaunaDB
 
-VSCode extension for FaunaDB.
+VS Code extension for FaunaDB.
 
-## Features
+## Getting Started
 
-### Browse your database data
+To get started using the Fauna VS Code extension, you'll need to set your secret key in order to access database information. Keys can be created in the Fauna Console webapp or via the Shell CLI.
```

Comment:
Link "Fauna Console webapp" to the URL: https://dashboard.fauna.com/

Reply:
@n400 Good call! Do you think there's any value in my adding a link to the Shell repo on GitHub as well?

Reply:
ahh yes.thank you :)